cmd/influx/telegrafs.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/influxdata/influx-cli/v2/clients/telegrafs"
+	"github.com/influxdata/influx-cli/v2/pkg/cli/middleware"
+	"github.com/urfave/cli/v2"
+)
+
+func newTelegrafsCommand() *cli.Command {
+	var params telegrafs.TelegrafParams
+	flags := append(commonFlags(), getOrgFlags(&params.OrgParams)...)
+	flags = append(flags, &cli.StringFlag{
+		Name:        "id",
+		Usage:       "Telegraf configuration ID to retrieve",
+		Aliases:     []string{"i"},
+		Destination: &params.Id,
+	})
+	return &cli.Command{
+		Name:  "telegrafs",
+		Usage: "List Telegraf configuration(s). Subcommands manage Telegraf configurations.",
+		Subcommands: []*cli.Command{
+			newCreateTelegrafCmd(),
+			newRemoveTelegrafCmd(),
+			newUpdateTelegrafCmd(),
+		},
+		Flags: flags,
```

Comment:
All the `cli.Command` structs in here need a `Before: middleware.WithBeforeFns(withCli(), withApi(true))` to ensure dependencies are properly set up. Otherwise `getAPI` and `getCLI` will panic at runtime.